elastic/beats
# add unit file states to system/service

## What does this PR do?

This PR adds the unit file state to `system/service` This tells you if the service is enabled or disabled, and what the vendor default is.

## Why is it important?

This is an important bit of data that appears in `systemctl status ...`

## Checklist



## How to test this PR locally

- Pull down PR and build
- run `system/service` on Linux, check to see if `unit_file` exists.

## Related issues

- https://github.com/elastic/beats/issues/22368
